plugins/a11ychecker.md

```diff
@@ -161,91 +295,212 @@ If `a11ychecker_allow_decorative_images` is set to `false`, the Accessibility Ch
 
 **Possible Values:** `true`, `false`
 
-#### Example: a11ychecker_allow_decorative_images
+#### Example: Using `a11ychecker_allow_decorative_images`
 
 ```js
 tinymce.init({
-    selector: 'textarea',
-    plugins: 'a11ychecker',
-    toolbar: 'a11ycheck',
-    a11ychecker_allow_decorative_images: true
+  selector: 'textarea',
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_allow_decorative_images: true
+});
+```
+
+### `a11ychecker_filter_issue`
+
+The `a11ychecker_filter_issue` option allows Accessibility Checker rules to be removed from the Accessibility Checker report using a callback function. This option is a flexible alternative to the `a11ychecker_ignored_rules` option.
```

Comment:
```suggestion
The `a11ychecker_filter_issue` option allows Accessibility Checker issues to be removed from the Accessibility Checker report using a callback function. This option is a flexible alternative to the `a11ychecker_ignored_rules` option.
```
since it specifically let's you remove violations in each report based on whatever info you want (e.g. rule id, element, etc.), not just removing specific rules?